docs/source/apidoc.md

```diff
@@ -133,13 +133,11 @@ See Ref. :cite:`Czarnik_2021_Quantum` for more details on these methods.
 .. automodule:: mitiq.interface.mitiq_qiskit.qiskit_utils
    :members:
 ```
-
-
 ## Classical Shadows
 
-### Classical Post-Processing
+### Classical Shadows (High-Level Tools)
```

Comment:
What if we remove this header and just have these functions directly under the "Classical Shadows" header? WDYT?